scripts/cvmfs/setup-releases.sh

```diff
@@ -153,6 +153,25 @@ elif [ "$os" = "ubuntu22.04" ]; then
     echo "Ubuntu 22.04 detected"
 fi
 
+_replace_marlin_dll() {
+    # replace the library on MARLIN_DLL with the local one (if any)
+    local pkg_name=${1}
+    local install_prefix=${2}
+    if echo ${MARLIN_DLL} | grep -qE "/${pkg_name}/"; then
+        local old_lib=$(echo ${MARLIN_DLL} | tr ":" "\n" | grep -E "/${pkg_name}/")
+        local lib_name=$(basename ${old_lib})
+        for d in lib lib64; do
```

Comment:
It seems none of the packages that appear in MARLIN_DLL install to lib64 so lib64 could not be checked for or not added preemptively